tests/src/reloadFunctionalTest/kotlin/org/jetbrains/compose/reload/tests/VerificationTests.kt

```diff
@@ -84,11 +84,60 @@ class VerificationTests {
         }
     }
 
+    @HotReloadTest
+    @QuickTest
+    fun `illegal code change - change local variables in main`(fixture: HotReloadTestFixture) = fixture.runTest {
+        val code = fixture.initialSourceCode(
+            """
+                import androidx.compose.material.Text
+                import org.jetbrains.compose.reload.test.*
+                
+                fun main() {
+                    val str1 = "Hello"
+                    screenshotTestApplication {
+                        Text(str1 + "world!")
```

Comment:
Please make sure to use 'TestText' instead of 'Text'.
'Text' might render slightly differently on different platforms or machines. 
TestText tries to be more consistent